### DISCUSSION TITLE
Running RV on Kaggle

### DISCUSSION THREAD

**Discussion by lcoandrade** [azavea/raster-vision#2002] [Q&A]
Hi there,

I know that the problem of running RV no Kaggle must be on the Kaggle side as seeing in the [issue](https://github.com/azavea/raster-vision/issues/1896). But is there something the development team could make to address this situation?

Running on Kaggle is really good, mostly for teaching purposes and for the datasets we can find on it.

Thanks in advance!

**Comment by AdeelH** [comment]
Does doing a "quick save" avoid the problem of Kaggle trying to re-run the notebook when saving?

**Reply by lcoandrade** [reply]
Yes, it does. It's a good workaround.
